native/core/src/execution/planner.rs

```diff
@@ -1220,7 +1217,7 @@ impl PhysicalPlanner {
                 // TODO: I think we can remove partition_count in the future, but leave for testing.
                 assert_eq!(file_groups.len(), partition_count);
 
-                let object_store_url = ObjectStoreUrl::local_filesystem();
+                let object_store_url = ObjectStoreUrl::parse("hdfs://namenode:9000").unwrap();
```

Comment:
this will be addressed in #1360

Reply:
The url should be available as part of the file path passed in. (see line 1178 above)

Reply:
Thanks @parthchandra it is already fixed.